requirements.txt

```diff
@@ -3,3 +3,4 @@ haversine
 httplib2
 -e git://github.com/pchampin/hydra-py.git#egg=hydra
```

Comment:
We don't need the `-e` flag. Please substitute these lines by using the new approach as displayed [here](https://github.com/HTTP-APIs/hydrus/blob/develop/requirements.txt#L26)